cmd/dep/status.go

```diff
@@ -49,6 +55,7 @@ func (cmd *statusCommand) Register(fs *flag.FlagSet) {
 	fs.BoolVar(&cmd.detailed, "detailed", false, "report more detailed status")
 	fs.BoolVar(&cmd.json, "json", false, "output in JSON format")
 	fs.StringVar(&cmd.template, "f", "", "output in text/template format")
+	fs.StringVar(&cmd.output, "o", "output.svg", "output file")
```

Comment:
Hah, so it was. That was in our spec, but we never actually implemented it, so I figured it wasn't there. Didn't bother to check!

So then, yes, it's simply a matter of dropping the `-o` param you added.